pkg/network/usm/shared_libraries.go

```diff
@@ -348,6 +394,7 @@ func (r *soRegistry) register(root, libPath string, pid uint32, rule soRule) {
 	r.m.Lock()
 	defer r.m.Unlock()
 	if _, found := r.blocklistByID[pathID]; found {
+		r.libBlocked.Add(1)
```

Comment:
shouldn't be increased here
we might see 1000 blocked libraries, although only 1 library caused it

Reply:
That depend what we want to detect
 * library that has been blocked but is used multiple times per new process/pid
 * library that failed to load, got only one hit per path

I added a telemetry HookFailed to cover both path